apstools/plans/alignment.py

```diff
@@ -329,8 +329,8 @@ def principal_signal_stats() -> str:
     def strong_peak(stats) -> bool:
         """Determine if the peak is strong."""
         try:
-            value = (stats.max_y - stats.min_y) / stats.sigma
-            return value > peak_factor
+            value = (stats.max_y - stats.min_y) / stats.stddev_y
+            return abs(value) > peak_factor
         except ZeroDivisionError:  # not enough samples
             try:
                 value = abs(stats.max_y / stats.min_y)
```

Comment:
Do we want to keep the format uniform across tries. Line 322 we return abs, line 336 we do the abs before return. Doesn't really matter in the grand scheme but I think it would be nice

Reply:
Good point.

Reply:
See commit 58d7230e